tests/DelegationSpec.hs

```diff
@@ -45,10 +46,11 @@ universeSize = UniverseSize
 
 spec :: Spec
 spec = do
-    runner   <- runIO createActionRunner
-    persist  <- runIO Persistent.mkRunPersistInMemory
-    uni      <- runIO $ unNat (runner persist) (mkUniverse universeSize)
-    snapshot <- runIO $ unNat (runner persist) getDBSnapShot
+    let catchAll (SomeException msg) = error . ("DelegationSpec.hs: " <>) . show $ msg
```

Comment:
then we couln't use `runIO'` elsewhere to throw something else, like the name of another test module.  or return, if we don't want to throw anything.